x-pack/plugins/snapshot_restore/server/routes/api/validate_schemas.ts

```diff
@@ -66,7 +66,7 @@ export const policySchema = schema.object({
 const fsRepositorySettings = schema.object({ location: schema.string() }, { unknowns: 'allow' });
 
 const fsRepositorySchema = schema.object({
-  name: schema.string(),
+  name: schema.string({ maxLength: 1000 }),
```

Comment:
I think this is outside the scope of this PR, but the error comes back a little weird I think based on how the `repository` schema is defined:

<img width="927" alt="Screenshot 2024-06-26 at 10 25 41 AM" src="https://github.com/elastic/kibana/assets/5226211/8f39418e-cd12-4068-955b-9df4a7c94a9f">